src/components/viewmodels/roomlist/RoomListViewModel.tsx

```diff
@@ -0,0 +1,18 @@
+/*
+Copyright 2025 New Vector Ltd.
+
+SPDX-License-Identifier: AGPL-3.0-only OR GPL-3.0-only OR LicenseRef-Element-Commercial
+Please see LICENSE files in the repository root for full details.
+*/
+
+import type { Room } from "matrix-js-sdk/src/matrix";
+import RoomListStoreV3 from "../../../stores/room-list-v3/RoomListStoreV3";
+
+export interface RoomListViewState {
+    rooms: Room[];
+}
+
+export function useRoomListViewModel(): RoomListViewState {
```

Comment:
Sorry, and this one too please, also on the current state would be good, as I assume it's going to grow more functionality later like subscribing to keep the list up to date.

Reply:
https://github.com/element-hq/element-web/pull/29357/commits/a7817e5e98c49d7cdaaa80b67da9bdc084289a4b ?